src/firmware_new/boards/frankie_v1/main.c

```diff
@@ -168,6 +171,16 @@ int main(void)
     MX_TIM4_Init();
     /* USER CODE BEGIN 2 */
 
+    //               ---- Initialize App/IO Layer ----
+    //
+    // At this point the UART peripheral should be configured correctly,
+    // so we initialize the logger with a UART robot log handler.
+    //
+    // Logs can been seen through `screen /dev/ttyACM0 115200`
+    io_uart_logger_init(&huart3);
```

Comment:
What does the 3 in huart3 mean?

Reply:
its just a name thats assigned by CubeMX, this is uart3 out of ~5 uart peripherals on the H7